README.md

```diff
@@ -1,2 +1,168 @@
-# gutenberg-license
-Stores the grants of consent and tooling for Gutenberg's dual license
+# Readme
+
+This repository holds the information relevant to the requests for consent from [Gutenberg's](https://github.com/WordPress/gutenberg) contributors to dual-license their past contributions to Gutenberg under the GPL and MPL (more information available here). For more information, you can look at the [initial discussion issue on GitHub](https://github.com/WordPress/gutenberg/issues/23651) as well as the [initial proposal post](https://make.wordpress.org/core/2020/09/21/proposal-dual-licensing-gutenberg-under-gpl-v2-0-and-mpl-v2-0/) on 21 September 2020, and [updated proposal post](https://make.wordpress.org/core/2021/03/05/dual-licensing-gutenberg-next-steps/) on Make/Core.
+
+# Format of `dual-license-responses.json`
+
+The [`dual-license-responses.json`](data/dual-license-responses.json) file records responses relevant to the work to update Gutenberg to be dual-licensed from (#31893)[https://github.com/WordPress/gutenberg/issues/31893] and (#31913)[https://github.com/WordPress/gutenberg/issues/31913].
```

Comment:
It looks like these links are not rendered properly. Since the `#` already makes a link out of these, would it make sense to remove the links?

Reply:
Oh, they're not rendering because I reversed the `()` and `[]`. I fixed that and I think it's good now. I don't think I can just use the #31893 text by itself to create the link because those are issues in a different repository, so GitHub won't connect them up.

Reply:
Makes sense 👍